stories/Components/UIcomponents/Author/Authorcolumn/Authors.jsx

```diff
@@ -1,67 +1,94 @@
+            <>
+              {Link == "False" ? (
+                <div className="author-wrapper__box" key={index}>
+                  {Image !== "False" && decNumber <= 3 ? (
+                    <Authorimg image={image} alt={alt} variant="Small" />
+                  ) : (<></>)}
+                  <div className="author__label">
+                    <Heading type="6" label={item.Authorlabel} />
+                    <P label={item.Authortext} />
+                  </div>
                 </div>
-              </div>
-            ))}
-          </>
-        )
-        : (
-          <div className="grid-x grid-margin-x">
-            {data.slice(0, decNumber).map((item, index) => (
-              <div className={['author-wrapper__box', 'cell', `${size}`].join(' ')} key={index}>
-                {(Image !== 'False') && (decNumber <= 3) ? (<Authorimg image={image} alt={alt} variant="Small" />) : (<></>)}
-                <div className="author__label">
-                  <Heading type="6" label={item.Authorlabel} />
-                  <P label={item.Authortext} />
+              ) : (
+                <a href="#" className="author-wrapper__box" key={index}>
+                  {Image !== "False" && decNumber <= 3 ? (
+                    <Authorimg image={image} alt={alt} variant="Small" />
+                  ) : (<></>)}
+                  <div className="author__label">
+                    <Heading type="6" label={item.Authorlabel} />
+                    <P label={item.Authortext} />
+                  </div>
+                </a>
+              )}
+            </>
+          ))}
+        </>
+      ) : (
+        <div className="grid-x grid-margin-x">
+          {data.slice(0, decNumber).map((item, index) => (
+            <>
+              {Link == "False" ? (
+                <div className={["author-wrapper__box", "cell", `${size}`].join(" ")} key={index}>
+                  {Image !== "False" && decNumber <= 3 ? (
+                    <Authorimg image={image} alt={alt} variant="Small" />
+                  ) : (<></>)}
+                  <div className="author__label">
+                    <Heading type="6" label={item.Authorlabel} />
+                    <P label={item.Authortext} />
+                  </div>
                 </div>
-              </div>
-            ))}
-          </div>
-        )}
+              ) : (
+                <a href="#" className={["author-wrapper__box", "cell", `${size}`].join(" ")} key={index}>
+                  {Image !== "False" && decNumber <= 3 ? (
+                    <Authorimg image={image} alt={alt} variant="Small" />
+                  ) : (<></>)}
+                  <div className="author__label">
+                    <Heading type="6" label={item.Authorlabel} />
+                    <P label={item.Authortext} />
+                  </div>
+                </a>
+              )}
+            </>
+          ))}
+        </div>
+      )}
     </>
   );
 };
```

Comment:
@Shubham-Sharma-77 seems like code is not linted as it should have single quotes and not double quotes